packages/esm-patient-registration-app/src/patient-registration/field/id/id-field.component.tsx

```diff
@@ -113,7 +113,13 @@ export const Identifiers: React.FC = () => {
 
   return (
     <div className={styles.halfWidthInDesktopView}>
-      <UserHasAccess privilege="coreapps.systemAdministration">
+      <UserHasAccess
+        privilege={[
+          'App: coreapps.addPatientIdentifierType',
+          'App: coreapps.deletePatientIdentifierType',
+          'App: coreapps.editPatientIdentifierType',
```

Comment:
I have marked the right place for these permissions, since the overlay only allows fetching all the identifier types and adding identifier types.

Also, I believe the correct priveleges should end with `identifier` and not `identifierType`, since we are not managing the identifier-types, but the saved identifiers instead.

Reply:
CC: @ibacher @reagan-meant @rbuisson

Reply:
@jona42-ui , please look into this asap.

Reply:
thanks @vasharma05 so that means i need to also conditionallly edit the handleEdit, handleReset functions to include the 
permission levels of users. it that right?

I believe what we are achieving here is allowing users have other privileges apart from deleting and then the admin should access all other privileges(did i get this right?)

Reply:
what i know is that;
**privilege** prop and only renders its child components if the user has the specified privilege.
corectme here if am not right pliz

Reply:
If you go to the patient registration form, you see a `configure` button. Clicking the Configure buttons opens an overlay.
<img width="433" alt="image" src="https://user-images.githubusercontent.com/51502471/221855416-ba8591af-40d5-4687-a143-036b7f6d8031.png">

As you can see, this allows adding identifiers to the patient's form. Hence the appropriate privileges for opening the configuring overlay are `coreapps.addPatientIdentifier`.

When you come to edit a patient, you can see the edit buttons and delete buttons:
<img width="404" alt="image" src="https://user-images.githubusercontent.com/51502471/221856685-3c3146af-fa60-4d61-8680-4715abe203d3.png">

Hence we need to place the appropriate privelege checks at appropriate places.

Reply:
So, see the preloaded privileges [here](https://dev3.openmrs.org/openmrs/admin/users/privilege.list). These just should be "Add Patient Identifiers" and "Edit Patient Identifiers" (removing an identifier should result in the identifier being voided, not deleted). I'm not sure where these other strings came from, but the idea is not to invent privileges but use the ones that are already being enforced on the backend.